From jbarnes@virtuousgeek.org  Fri Apr 18 14:57:07 2008
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 17 Apr 2008 10:22:27 -0700
Subject: PCI: Change PCI subsystem MAINTAINER
Cc: Jesse Barnes <jbarnes@virtuousgeek.org>
Message-ID: <200804171022.27529.jbarnes@virtuousgeek.org>
Content-Disposition: inline

Jesse foolishly volunteered to handle PCI patches.  Update MAINTAINERS to
reflect this.

Woho!  Time to kick back and relax...

Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3068,11 +3068,10 @@ L:	linux-pci@atrey.karlin.mff.cuni.cz
 S:	Supported
 
 PCI SUBSYSTEM
-P:	Greg Kroah-Hartman
-M:	gregkh@suse.de
+P:	Jesse Barnes
+M:	jbarnes@virtuousgeek.org
 L:	linux-kernel@vger.kernel.org
 L:	linux-pci@atrey.karlin.mff.cuni.cz
-T:	quilt kernel.org/pub/linux/kernel/people/gregkh/gregkh-2.6/
 S:	Supported
 
 PCI HOTPLUG CORE
